cardano-cli/hie.yaml

```diff
@@ -0,0 +1,13 @@
+cradle:
```

Comment:
Doesn't the  next commit add this to `.gitgnore` ? If so, this file should be droppped from this commit.

Reply:
Hmm, yeah.  I need to re-order my commits.